app/forms/decidim/reporting_proposals/admin/proposal_photo_form.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ReportingProposals
+    module Admin
+      class ProposalPhotoForm < Decidim::Form
+        include Decidim::AttachmentAttributes
+        attribute :attachment, AttachmentForm
+        attachments_attribute :photos
+      end
```

Comment:
```suggestion
        def current_component
          @current_component ||= @proposal&.component
        end
      end
```
Because our controller is not linked in the participatory space as a subcontroller, it does not have certain properties that a `ComponentController`has (like the current_component method). So, here in the form we can extract the same information frrom the proposal itself.